### DISCUSSION TITLE
about search

### DISCUSSION THREAD

**Discussion by ccczhouxin** [squidfunk/mkdocs-material#4489] [Q&A]
How can i cancel timely search,i want input a key then delete ,and the result still in the page .
example:
![image](https://user-images.githubusercontent.com/43168824/195032972-25e7d205-dc1e-4ef5-a4b5-8881010ac4b5.png)

**Comment by squidfunk** [comment]
This seems not at all related to Material for MkDocs...

**Reply by ccczhouxin** [reply]
Hello,that a example,i want  input a value to seach "1",then i delete the value,the result still in the screen,not blank
![image](https://user-images.githubusercontent.com/43168824/195295662-5d7a7676-4f2b-4e55-ab2c-3e7aa02686da.png)
![image](https://user-images.githubusercontent.com/43168824/195295558-5b1eb866-c989-4746-b338-7e44e03c8e54.png)
